### DISCUSSION TITLE
"It will output" statements in documentation

### DISCUSSION THREAD

**Discussion by gputzel** [nextflow-io/nextflow#2759] [Q&A] [answered]
As a newcomer, I am confused by some statements in the Nextflow documentation regarding the outputs of processes. For example, the subsection about [input of type stdin](https://www.nextflow.io/docs/latest/process.html#input-of-type-stdin) gives the following example:

```
str = Channel.from('hello', 'hola', 'bonjour', 'ciao').map { it+'\n' }

process printAll {
   input:
   stdin str

   """
   cat -
   """

}
```

and says

> It will output:
> `hola`
> `bonjour`
> `ciao`
> `hello`

If I run this workflow, I see no output, and I realize that this is as expected given the way processes work. However, the statement still strikes me as inaccurate and confusing for newcomers to Nextflow. Am I correct or does this just mean I have not yet quite internalized the process/channel model?

**Comment by pditommaso** [comment, marked as answer]
Yeah, I agree, it's a bit tricky. Nextflow does not show the task console output by default. You should type the following command see it 

```
nextflow run <YOUR SCRIPT> -process.echo 
```

**Reply by gputzel** [reply]
Thank you. I wasn't aware of this command line option. I notice that some of the later example processes in the documentation include the `echo true` directive. It might be worth adding that directive to all of the examples, at least where the text states that they output something. That way, people learning Nextflow can run the examples and see the same output.
